.changeset/afraid-suits-cry.md

```diff
@@ -0,0 +1,16 @@
+---
+"@khanacademy/kas": patch
+"@khanacademy/keypad-context": patch
+"@khanacademy/kmath": patch
+"@khanacademy/math-input": patch
+"@khanacademy/perseus": patch
+"@khanacademy/perseus-core": patch
+"@khanacademy/perseus-editor": patch
+"@khanacademy/perseus-linter": patch
+"@khanacademy/perseus-score": patch
+"@khanacademy/perseus-utils": patch
+"@khanacademy/pure-markdown": patch
+"@khanacademy/simple-markdown": patch
+---
```

Comment:
I'm patch-bumping all of our packages even though they have no production changes. This will mean that after this release, all packages will document provenance on npmjs.com from here on forward.